extensions/amp-jwplayer/0.1/amp-jwplayer.js

```diff
@@ -446,6 +449,14 @@ class AmpJWPlayer extends AMP.BaseElement {
           this.playlistItem_ = playlistItem;
           this.sendCommand_('getPlayedRanges');
           break;
+        case 'resize':
+          const {width, height} = detail;
+          Services.mutatorForDoc(this.getAmpDoc()).forceChangeSize(
```

Comment:
Forcing a size change is a rarity in AMP components. Can we instead use `attemptChangeSize`?

Reply:
I've tried this in a few different ways to no avail (wouldn't resize with attemptChangeSize, no errors/warnings), but this seemed to work seamlessly. Is this something you've seen in the past? If so, how was it resolved?

Reply:
This is UX policy for AMP. Resizing and other operations that can cause content jumping may only be done off-screen or if triggered by a user gesture.